src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/FxCopAnalyzersPackage.cs

```diff
@@ -7,8 +7,14 @@
 namespace FxCopAnalyzersSetup
 {
     [Guid(PackageGuid)]
+    // Package is used by VS
+#pragma warning disable CA1812 // Avoid uninstantiated internal classes
```

Comment:
>#pragma warning disable CA1812 // Avoid uni [](start = 0, length = 43)

This is very messy and cluttered. I can't see the actual code anymore. Can you move the pragmas to the top of the file?

```C#
#pragma warning disable CA1812 
#pragma warning disable CA1823
```

There is no need to need to restoring these.

Reply:
Done.